src/index.html

```diff
@@ -9,10 +9,51 @@
     <title>Stars</title>
     <link
       rel="stylesheet"
-      href="./style.css"
+      href="./blocks/style.css"
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars__str"></div>
+      <div class="stars__str"></div>
+      <div class="stars__str"></div>
+      <div class="stars__str"></div>
+      <div class="stars__str"></div>
+    </div>
```

Comment:
consider using `ul` + `li` for the list of the stars


rename `str` to `star`